src/ogc-api/endpoint.ts

```diff
@@ -245,4 +245,69 @@ export default class OgcApiEndpoint {
       })
       .then(fetchDocument<OgcApiCollectionItem>);
   }
+
+  /**
+   * Asynchronously retrieves a URL for the items of a specified collection, with optional query parameters.
+   * @param collectionId - The unique identifier for the collection.
+   * @param options - An object containing optional parameters:
+   *  - query: Additional query parameters to be included in the URL.
+   *  - outputFormat: The MIME type for the output format. Default is 'json'.
+   *  - maxFeatures: The maximum number of features to include.
+   *  - skipGeometry: Flag to skip geometry data in the response.
+   *  - extent: Bounding box to limit the features.
+   *  - offset: Pagination offset for the returned results.
+   *  - outputCrs: Coordinate Reference System code for the output.
+   *  - extentCrs: Coordinate Reference System code for the bounding box.
+   *  - properties: An array of property names to be returned.
+   * @returns A promise that resolves to the URL as a string or rejects if an error occurs.
+   */
+  getCollectionItemsUrl(
+    collectionId: string,
+    options: {
+      query?: string;
+      outputFormat?: string;
+      maxFeatures?: number;
```

Comment:
In OGC API we talk about "items" instead of features; how about simply calling this option "limit"? It is a very usual term for pagination